packages/backend/src/index.ts

```diff
@@ -1,25 +1,39 @@
+import { parseEnv } from './config/envParser.js'
+parseEnv()
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const config = require('../knexfile.js')
+
+import cookieParser from 'cookie-parser'
 import cors from 'cors'
-import * as dotenv from 'dotenv'
 import express, { Application } from 'express'
 import Knex from 'knex'
 import { Model } from 'objection'
 import passport from 'passport'
-dotenv.config()
+import { jwtStrategy } from './auth/jwtStrategy.js'
+
+import { errorHandler } from './middlewares/errorHandler'
+import { mainRouter } from './routes.js'
 
 const app: Application = express()
 
 app.disable('X-Powered-By')
 
 app.use(cors())
 app.use(express.json())
+app.use(express.urlencoded({ extended: true }))
+app.use(cookieParser())
 
 const knex = Knex(config[process.env.NODE_ENV || 'development'])
 Model.knex(knex)
 
 app.use(passport.initialize())
 
+passport.use(jwtStrategy)
+
+app.use(mainRouter)
+
+app.use(errorHandler)
+
 app.listen(process.env.PORT, (): void => {
```

Comment:
If we are going with the changes mentioned above, we should import the `env` object and change this to:
```suggestion
app.listen(env.PORT, (): void => {
```

Reply:
Ditto for every `process.env.VARIABLE`.